library/ssl_tls13_client.c

```diff
@@ -1560,10 +1673,6 @@ int mbedtls_ssl_tls13_handshake_client_step( mbedtls_ssl_context *ssl )
             ret = ssl_tls13_read_server_hello( ssl );
             break;
 
-        case MBEDTLS_SSL_ENCRYPTED_EXTENSIONS:
```

Comment:
Please do not change it. to avoid conflict

Reply:
Change it back. Done